Bottom: f48e90adb114d1d92b8cb1ba5cc26e22d0eba9c1
Top:    68f8c9e4bacadf9557668af3d789ced377646cde
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-21 00:10:16 +0100

Refresh of crosshair-c-use-a-dummy-via-st

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 87e45fb..f1e3ce7 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -94,7 +94,7 @@ thindraw_moved_pv (PinType *pv, int x, int y)
   moved_pv.X += x;
   moved_pv.Y += y;
 
-  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, false, false);
+  gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &moved_pv, true, false);
 }
 
 /* ---------------------------------------------------------------------------
@@ -574,7 +574,7 @@ DrawAttached (void)
         via.Flags = NoFlags ();
         TOGGLE_FLAG (SQUAREFLAG, &via);
 
-        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, true, false);
+        gui->thindraw_pcb_pv (Crosshair.GC, Crosshair.GC, &via, false, false);
 
         if (TEST_FLAG (SHOWDRCFLAG, PCB))
           {
